src/engine.js

```diff
@@ -98,36 +63,71 @@ const editTaskDescription = (index, newDescription) => {
     saveTasksToLocalStorage();
     displayItemsByIndex();
   } else {
-    alert('Invalid task index.');
+    alert('please add a task to init app');
```

Comment:
- [ ] **[OPTIONAL]** For a better user experience it could be easier to add the task by clicking Enter.